src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,41 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
```

Comment:
Ensure that the `getUser` function is correctly imported from the `api` module. Verify that this module exists and properly exports the function.